replay_build_scripts/devtools-tests.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+
+set -eu
+
+if [ -n "${BUILDKITE-}" ]; then
+    buildkite-agent artifact download build_id/linux/x86_64/build_id ./
+    BUILD_ID=$(cat build_id/linux/x86_64/build_id)
+
+    if [ -z "${BUILD_ID}" ]; then
+        echo "Unable to retrieve build id from buildkite agent"
+        exit 1
+    fi
+    RUN_ID="$BUILDKITE_PIPELINE_SLUG/$BUILDKITE_BUILD_NUMBER"
+else
+    BUILD_ID=$1
+    if [ -z "${BUILD_ID}" ]; then
+        echo "build ID required as first argument";
+        exit 1
+    fi
+    RUN_ID=test-$(uuidgen)
+fi
+
+if [ -z "${GITHUB_AUTH_SECRET}" ]; then
+    echo "GITHUB_AUTH_SECRET is not set in environment"
+    exit 1
+fi
+
+
+# TODO(toshok) when/if the devtools PR (https://github.com/replayio/devtools/pull/10001) lands, change the ref here to `main`
+INPUTS='{"ref":"toshok/accept-run-id","inputs":{"runId":"'${RUN_ID}'","linuxBuildFile":"'${BUILD_ID}'.tar.xz"}}'
```

Comment:
leaving this TODO in for the time being just in case I need more testing.  The sequence will be:

1. land devtools PR
2. change this PR to ref `main`, make sure things still work.
3. land this PR.

Reply:
devtools PR landed, and have updated this.